src/engine/devlinks.rs

```diff
@@ -17,104 +17,153 @@ use super::engine::DEV_PATH;
 
 /// Set up the root Stratis directory, where dev links as well as temporary
 /// MDV mounts will be created. This must occur before any pools are setup.
-pub fn setup_dev_path() -> StratisResult<()> {
+pub fn setup_dev_path() -> () {
```

Comment:
Maybe we leave this returning a result? If this fails, this isn't just the user futzing around with symlinks, this means stratisd has some sort of permission problem, and it might be better to bail.